data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/export/DataFileLoader.java

```diff
@@ -37,30 +46,37 @@ public class DataFileLoader implements Runnable {
      */
     private static final int DEFAULT_CHECKPOINT_INTERVAL_MILLS = 2 * 60_000;
 
+    static final Duration BUFFER_TIMEOUT = Duration.ofSeconds(60);
+    static final int DEFAULT_BUFFER_BATCH_SIZE = 1_000;
```

Comment:
how were this number chosen ?

Reply:
Both number won't make too much differences. 
There will be retries after timeout anyway. 1000 is the batch size we used for reading file and reading stream data, so we just use the same.